odm/pom.xml

```diff
@@ -1,76 +1,103 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.libreclinica.odm</groupId>
-    <artifactId>LibreClinica-odm</artifactId>
-    <packaging>jar</packaging>
-    <version>2.2</version>
-    <name>LibreClinica ODM</name>
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-    <dependencies />
-    <build>
-        <plugins>
-            <!-- Automate the creation of binding classes for XML schemas - JAXB marshall/ unmarshall -->
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>jaxb2-maven-plugin</artifactId>
-                <version>2.3</version>
```

Comment:
Although it was present already here the version definition should be moved to the maine pom.xml.